### DISCUSSION TITLE
Request to add ECS support

### DISCUSSION THREAD

**Discussion by liang-hiwin** [semihalev/sdns#315] [Ideas]
Request to add ECS support https://en.wikipedia.org/wiki/EDNS_Client_Subnet

**Comment by semihalev** [comment]
There were support this but I disabled for privacy reasons.

**Reply by liang-hiwin** [reply]
> There were support this but I disabled for privacy reasons.

ECS can improve the accuracy of CDN domain names

**Comment by semihalev** [comment]
Implemented and will be release on v1.7.0
